pkg/policy/mapstate.go

```diff
@@ -807,7 +894,7 @@ func (mc *MapChanges) consumeMapChanges(policyMapState MapState) (adds, deletes
 			// insert but do not allow non-redirect entries to overwrite a redirect entry,
 			// nor allow non-deny entries to overwrite deny entries.
 			// Collect the incremental changes to the overall state in 'mc.adds' and 'mc.deletes'.
-			policyMapState.denyPreferredInsertWithChanges(mc.changes[i].Key, mc.changes[i].Value, adds, deletes)
+			policyMapState.denyPreferredInsertWithChanges(mc.changes[i].Key, mc.changes[i].Value, adds, deletes, nil)
```

Comment:
I presume that you plan to look closer into these `nil` cases? I'll skip thinking about them for now but I suspect they'll need to be populated somehow.